packages/calendar/src/index.vue

```diff
@@ -98,6 +100,7 @@ export default defineComponent({
 
   emits: ['input', 'update:modelValue'],
   setup(props, ctx) {
+    const locale = useLocaleInject()
```

Comment:
Just a small problem, why don‘t we use `const { t } = useLocaleInject()` instead of `const locale = useLocaleInject()`?

So we don't even have to modify the code we already have.

```
- return `${date.value.year()} ${t('el.datepicker.year')} ${t(pickedMonth)}`
+ return `${date.value.year()} ${locale.t('el.datepicker.year')} ${locale.t(pickedMonth)}`
```

Reply:
Since useLocaleInject returns a `reactive` instead of a `ref`

Reply:
May be we can make it be same as this form of `vue-i18n` API.

> <https://github.com/intlify/vue-i18n-next/blob/1d7843afad0aa1fae51728dec1505e19b5d1058f/examples/composition/started.html#L47>

Reply:
ur right, I'm going to update the code to this style